Documentation/Grid/Technical.md

```diff
@@ -27,4 +27,21 @@ Each tile has specified:
 WM and BM:
 texture
 flip_h
-type (string for game logic)
\ No newline at end of file
+type (string for game logic)
+
+
+
+## Directions
+We start looking toward the left then we move clockwise
```

Comment:
nit: change to ascii art

Reply:
done, also cleaned up the doc
to preview click [here](https://github.com/Wiolarz/Krong/blob/Polyspear_world_fix_battle/Documentation/Grid/Technical.md)